letta/server/ws_api/server.py

```diff
@@ -2,20 +2,42 @@
 import signal
 import sys
 import traceback
+import os
+from typing import Optional
 
 import websockets
 
 import letta.server.ws_api.protocol as protocol
 from letta.server.constants import WS_DEFAULT_PORT
 from letta.server.server import SyncServer
 from letta.server.ws_api.interface import SyncWebSocketInterface
+from letta.server.webhook.factory import create_streaming_interface_from_env
 
 
 class WebSocketServer:
-    def __init__(self, host="localhost", port=WS_DEFAULT_PORT):
+    def __init__(self, host="localhost", port=WS_DEFAULT_PORT, webhook_url: Optional[str] = None, webhook_token: Optional[str] = None):
         self.host = host
         self.port = port
-        self.interface = SyncWebSocketInterface()
+        self.webhook_url = webhook_url or os.getenv("LETTA_WEBHOOK_URL")
+        self.webhook_token = webhook_token or os.getenv("LETTA_WEBHOOK_TOKEN", "")
```

Comment:
```suggestion
        self.webhook_url = webhook_url
        self.webhook_token = webhook_token
```
you already have checked the `LETTA_WEBHOOK_URL` and `LETTA_WEBHOOK_TOKEN` on the `start_server` method

Reply:
also `LETTA_WEBHOOK_TOKEN` and `LETTA_WEBHOOK_URL` is used in multiple places, you may put them in one place like `webhook/constants.py` like in https://github.com/letta-ai/letta/blob/e4a7bb1489392142c7cdc4b90f87d1cbff999b93/letta/constants.py#L1-L21

Reply:
Thanks for the contribution @ahmedrowaihi